src/azure-cli/azure/cli/command_modules/iot/_params.py

```diff
@@ -281,9 +281,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                    help='Location of your IoT Central application. Default is the location of target resource group.')
         c.argument('sku', arg_type=get_enum_type(AppSku),
                    help='Pricing tier for IoT Central applications. Default value is ST2.')
-        c.argument('subdomain',
+        c.argument('subdomain', options_list=['--subdomain', '--s'],
```

Comment:
shouldn't it be -s instead of --s? (same thing with below)

Reply:
I'm just follow iot hub, https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/iot/_params.py#L125, but single dash also make sense

Reply:
I think if it is one word then it should be -n (for example name)
so, since subdomain is one word, it should probably be -s
template is also one word, so it should probably be -t
display name is two words, so what you have seems correct
let me know what your thoughts on this